openedx_webhooks/templates/github_community_pr_comment.md.j2

```diff
@@ -1,52 +1,100 @@
-{% filter replace("\n", " ")|trim %}
-Thanks for the pull request, @{{ user }}!
-Please note that it may take us up to several weeks or months to complete a review and merge your PR.
-{% endfilter %}
-
-Feel free to add as much of the following information to the ticket as you can:
-- supporting documentation
-- [Open edX discussion forum threads](https://discuss.openedx.org/)
-- timeline information ("this must be merged by XX date", and why that is)
-- partner information ("this is a course on edx.org")
-- any other information that can help Product understand the context for the PR
-
-{% filter replace("\n", " ")|trim %}
-All technical communication about the code itself will be done via the
-GitHub pull request interface. As a reminder,
-[our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).
-{% endfilter %}
+Thanks for the pull request, `@{{ user }}`!
```

Comment:
why is the user in back ticks now? i see that it works ok without them right now.

Reply:
I'm not entirely sure, but I've taken the contents from [here](https://gist.github.com/itsjeyd/a311f90c28dae77d2f5fddd72ebd47b7). 

Generally, the idea with putting it in backticks is to avoid actually pinging the user. However, in this case perhaps the user should be notified? 

@itsjeyd Should the backticks be here? I think not notifying the maintainer group or user makes sense but the PR author perhaps should be OK to ping?

Reply:
@xitij2000 @DanielVZ96 I don't remember why I added those backticks, feel free to remove them 😅

Reply:
@xitij2000 @itsjeyd i also prefer the ping :sweat_smile: